src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -357,6 +359,29 @@ def _store_testcase_for_regression_testing(testcase, testcase_file_path):
     # Only store crashes with bugs associated with them.
     return
 
+  if not progression_task_input.HasField('regression_testcase_url'):
+    # No work to do.
+    return
+
+  regression_testcase_url = progression_task_input.regression_testcase_url
+
+  with open(testcase_file_path, 'rb') as testcase_file_handle:
+    testcase_file = testcase_file_handle.read()
+    if storage.upload_signed_url(testcase_file, regression_testcase_url):
```

Comment:
upload_signed_url will return True or exception, so it probably shouldn't return anything at all (if you want to be a hero you can change this in storage.py) so let's not depend on this behavior.